Gx cooldown 4   deprecations and be api update (#22)
MewsSystems/gitbook-booking-engine: FAQ/README.md

* In the upper section there ia a field **City Identifier**. This is your City ID. It has a format `aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee`.

![City ID in Mews Operations](../.gitbook/assets/commander-city-id.png)

> ## Where can I get age category ID?

Reviewer: ```suggestion
## Where can I get Age Category ID?
```
Just for consistency with `Configuration ID` and `City ID`.

You can find the AgeCategory ID in **Mews Operations**:

* Start on the **Dashboard** (the main screen).
* In the left-hand menu, select **Settings** -> **Services**.
* In the section **Bookable services**, select the service which has the Booking Engine configuration. If there's just one service, select that one.
* On the left, select **Age categories**.
* Select the age category you want.

> * In the URL there is an ID in the end, right after `/Detail/`. This is your age category ID. It has a format `aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee`.

Reviewer: ```suggestion
* In the URL there is an ID at the end, right after `/Detail/`. This is the age category ID. It has the format `aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee`.
```

> 

Reviewer: It's a small point, but I would remove the word DEVELOPMENT from the image (where the screnshot is taken from the Development Server), what do you think?
